.bazelrc

```diff
@@ -117,8 +117,7 @@ build:dynamic_kernels --define=dynamic_loaded_kernels=true
 build:dynamic_kernels --copt=-DAUTOLOAD_DYNAMIC_KERNELS
 
 # Build TF with C++ 17 features.
-build:c++17 --cxxopt=-std=c++1z
-build:c++17 --cxxopt=-stdlib=libc++
+build:c++17 --cxxopt=-std=c++17
```

Comment:
This also changes the default lib, the assumption is that we want those decisions to be orthogonal?

Reply:
GCC doesn't support the `-stdlib` command line flag so this broke when building tensorflow with c++17 support. If we want to be able to choose a different standard library I think it will need to be a different bazel flag.